app/assets/javascripts/events.coffee

```diff
@@ -1,14 +1,16 @@
 $(document).ready ->
```

Comment:
I know that this line is not related to your PR but please rewrite it using coffee ok? :)
```coffee
jQuery () ->
```
or 
```coffee
$ () ->